forge-gui/res/cardsfolder/upcoming/irenicuss_vile_duplication.txt

```diff
@@ -0,0 +1,5 @@
+Name:Irenicus's Vile Duplication
+ManaCost:3 U
+Types:Sorcery
+A:SP$ CopyPermanent | Cost$ 3 U | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | AddKeywords$ Flying | NonLegendary$ True | SpellDescription$ Create a token that's a copy of target creature you control, except the token has flying and isn’t legendary if that creature is legendary.
```

Comment:
`Cost$` param is not needed on SP$ lines if it matches `ManaCost`

`isn’t` -> `isn't`

Reply:
Removed redundant cost param

Corrected ASCII apostrophe char in spell description and oracle text